src/test/regress/expected/multi_create_table.out

```diff
@@ -173,14 +182,89 @@ SELECT master_create_distributed_table('unique_const_on_two_columns', 'partition
  
 (1 row)
 
+CREATE TABLE exclusion_const_on_part_col
+(
+	partition_col integer,
+	other_col integer,
+	CONSTRAINT exclusion_const_on_part_col_excl EXCLUDE (partition_col WITH =)
+);
+SELECT master_create_distributed_table('exclusion_const_on_part_col', 'partition_col', 'hash');
+ master_create_distributed_table 
+---------------------------------
+ 
+(1 row)
+
+SELECT master_create_worker_shards('exclusion_const_on_part_col', '2', '1');
+ master_create_worker_shards 
+-----------------------------
+ 
+(1 row)
+
+INSERT INTO exclusion_const_on_non_part_col (partition_col, other_col) VALUES (1,1);
```

Comment:
I guess these `INSERT`s should go to `exclusion_const_on_part_col` and we'd see errors?